docs/migration-guides/0.8-0.9.md

```diff
@@ -0,0 +1,37 @@
+## Migrate from v0.8 to v0.9
```

Comment:
We agreed on the following attempt to migration guides: 
Include instructions for components that need to be handled manually by sys admins. If a component can be migrated automatically with no admin action required, do not write anything about it. 

Therefore in your case you should add an H2 heading with the component name to make it clear that this component needs admins attention. 

`Migrate from v0.8 to v0.9` should be the main heading of the document (H1) and under it the basic premise of this doc (if a component is not described - no action required, if described - follow the instructions) should be described.